db/migrate/20190815151449_add_verify_response_to_claim.rb

```diff
@@ -0,0 +1,5 @@
+class AddVerifyResponseToClaim < ActiveRecord::Migration[5.2]
+  def change
+    add_column :claims, :verify_response, :json, null: false, default: "{}"
```

Comment:
Does it make sense to be setting a default here? Doesn't it make more sense for it to default to null so we can tell the difference between those who have been through Verify, and those who haven't?

Reply:
From what I remember, if you don't set a default in JSON columns, it can cause unexpected behaviour

Reply:
@pezholio can you confirm this? In my mind we shoudn't need a default, unless we're planning on running queries against the JSON, which I can't see us needing to do here.

Reply:
Yeah, I've removed it and it seems fine. I guess the only time it would cause problems would be if we were querying the JSON